README.rst

```diff
@@ -66,7 +66,6 @@ The following diagram shows the available subcommands of zazu.
     "repo" -> "repo_init"
```

Comment:
You need to merge with latest develop. I removed the pngs and replaced them with a call to a service that renders them.